.github/workflows/cross-merge.yml

```diff
@@ -0,0 +1,79 @@
+name: Cross merge
+
+on:
+  workflow_call:
+    secrets:
+      robot-token:
+        description: 'A PAT token used by the app/robot'
+        required: true
+
+jobs:
+  job:
```

Comment:
:heart: Would love a proper name

Reply:
It's fine if it's in a reusable state, IMHO.
<img width="454" alt="image" src="https://user-images.githubusercontent.com/67897517/144590032-186eddb8-fcb7-4735-9872-b284d0db7d61.png">

Reply:
Not in my opinion. If it were `Ibexa Reusable / Cross Org PR` it would be immediately obvious what it's for.

Reply:
OK, because the comment seemed to relate to line 11.